src/atomate2/abinit/sets/base.py

```diff
@@ -119,12 +252,27 @@ def validate(self) -> bool:
             ext = fname2ext(in_file)
             if ext is None:
                 return False
+            # Need to consider irdddk to read 1WF files
+            # if ext == "1WF" and (
+            #     "ird1wf" in self.abinit_input or "irdddk" in self.abinit_input
+            # ):
+            #     return (
+            #         self.abinit_input["ird1wf"] == 1 or
+            # self.abinit_input["irdddk"] == 1
+            #     )
             irdvars = irdvars_for_ext(ext)
+            if ext == "1WF":
+                irdvars["irdddk"] = 1
             for irdvar, irdval in irdvars.items():
-                if irdvar not in self.abinit_input:
-                    return False
-                if self.abinit_input[irdvar] != irdval:
-                    return False
+                if irdvar in self.abinit_input and self.abinit_input[irdvar] == irdval:
+                    # at least one irdvar has the right irdval in abinit_input
+                    break
```

Comment:
Here the logic seems wrong. 
This means that if there is more than one dependency in `irdvars`, if the first is fine, the second will not be checked. I think the old implementation was fine as it was.